straxen/plugins/veto_events.py

```diff
@@ -0,0 +1,286 @@
+        n_events = len(events)
+        events[self.name_event_number] = np.arange(n_events) + self.events_seen
+        self.events_seen += n_events
+
+        return events
+
+
+def veto_event_dtype(name_event_number='event_number_nv', n_pmts=120):
+    dtype = []
+    dtype += strax.time_fields  # because mutable
+    dtype += [(('Veto event number in this dataset', name_event_number), np.int64),
+              (('Last hitlet endtime in event [ns].', 'last_hitlet_endtime'), np.int64),
+              (('Total area of all hitlets in event [pe]', 'area'), np.float32),
+              (('Total number of hitlets in events', 'n_hits'), np.int32),
+              (('Total number of contributed channels', 'n_contributed_pmt'), np.int32),
+              (('Area in event per channel [pe]', 'area_per_channel'), np.float32, n_pmts),
+              (('Area weighted mean time of the event relative to the event start [ns]',
+                'center_time'), np.float32),
+              (('Weighted variance of time [ns]', 'center_time_spread'), np.float32),
+              ]
+    return dtype
+
+
+@numba.njit(cache=True, nogil=True)
+def compute_nveto_event_properties(events, contained_hitlets, start_channel=2000):
+    """
+    Computes properties of the neutron-veto events. Writes results
+    directly to events.
+
+    :param events: Events for which properties should be computed
+    :param contained_hitlets: numba.typed.List of hitlets contained in
+        each event.
+    :param start_channel: Integer specifying start channel, e.g. 2000
+        for nveto.
+    """
+    for e, hitlets in zip(events, contained_hitlets):
+        event_area = np.sum(hitlets['area'])
+        e['area'] = event_area
+        e['n_hits'] = len(hitlets)
+        e['n_contributed_pmt'] = len(np.unique(hitlets['channel']))
+
+        t = hitlets['time'] - hitlets[0]['time']
+        if event_area:
+            e['center_time'] = np.sum(t * hitlets['area']) / event_area
+            if e['n_hits'] > 1 and e['center_time']:
+                w = hitlets['area']/e['area'] # normalized weights
+                # Definition of variance
+                e['center_time_spread'] = np.sqrt(np.sum(w*np.power(t-e['center_time'],2))/np.sum(w))
+            else:
+                e['center_time_spread'] = np.inf
+
+        # Compute per channel properties:
+        for h in hitlets:
+            ch = h['channel'] - start_channel
+            e['area_per_channel'][ch] += h['area']
+
+        # Compute endtime of last hitlet in event:
+        endtime = strax.endtime(hitlets)
+        e['last_hitlet_endtime'] = max(endtime)
+
+
+@strax.takes_config(
+    strax.Option('angle_max_time_nv', default=10,
+                 help="Time [ns] within an evnet use to compute the azimuthal angle of the event."),
+    strax.Option('nveto_pmt_position_map', track=False,
+                 help="nVeto PMT position mapfile",
+                default='nveto_pmt_position_mc.csv'),
```

Comment:
Could you please change the default file name to `nveto_pmt_position.csv`?

Reply:
```suggestion
                default='nveto_pmt_position.csv'),
```